plot_pixel_values.py

```diff
@@ -22,41 +24,103 @@ def get_opt():
     parser.add_argument('--fit-scaling',
                         action='store_true',
                         help='Fit the dark and scale factor (default=False)')
-
+    parser.add_argument('--plot-fits',
```

Comment:
Also, it is presently the easiest place to see the range of CCD temperatures.  Where did you want to plot those again?